### ISSUE TITLE
[Bug] api-ms-win-core-winrt-l1-1-0.dll on Win 7

### ISSUE THREAD

**Issue by wvxwxvw** [NickeManarin/ScreenToGif#1077]
- OS: Win 7 x64 sp1 + all update
 - Version 2.36 portable x64 full
   
ScreenToGif.exe - System error
  
The program cannot be started because there is no api-ms-win-core-winrt-l1-1-0.dll . Try reinstalling the program.

**Comment by NickeManarin** [comment]
Can you still run the light variant? 
The one that doesn't come with .NET 6 runtime.

**Comment by NickeManarin** [comment]
Also, can you check the event log for the stack trace?

**Comment by wvxwxvw** [comment]
I remove net 6.
I put in the folder api-ms-win-core-winrt-l1-1-0.dll and the program started working.
https://ru.dll-files.com/download/95cfee0b406bd20c57333653af3a45fc/api-ms-win-core-winrt-l1-1-0.dll.html?c=MUpldm51MGFieFYzVE5BQTljQnBrQT09
  
There are no errors, only due to the absence of .net, after I tried to launch the light version for some reason.
![2022 02 12_233932](https://user-images.githubusercontent.com/13194155/153727677-c4ccd6c6-1128-4666-ad93-1f30922fc1da.png)

**Comment by NickeManarin** [comment]
https://github.com/dotnet/core/issues/6976

**Comment by NickeManarin** [comment]
Minimum target is set to Windows 7 on each assembly's properties.
So the export should have worked, unless there's other missing .NET 6 requirements.

**Comment by wvxwxvw** [comment]
Yes, I understand. My launch of the light version is irrelevant. You need to integrate this dll. That's it.
	
And by the way, when installing the previous version, with pre-installation .net 6, the program started without this dll.
